prow/plugins/golint/golint.go

```diff
@@ -56,18 +59,28 @@ func handleIC(pc plugins.PluginClient, ic github.IssueCommentEvent) error {
 }
 
 // modifiedGoFiles returns a map from filename to patch string for all go files
-// that are modified in the PR excluding vendor/.
-func modifiedGoFiles(ghc githubClient, org, repo string, number int) (map[string]string, error) {
+// that are modified in the PR excluding vendor/ and generated files.
+func modifiedGoFiles(ghc githubClient, org, repo string, number int, sha string) (map[string]string, error) {
 	changes, err := ghc.GetPullRequestChanges(org, repo, number)
 	if err != nil {
 		return nil, err
 	}
 
+	gfg, err := genfiles.NewGenFilesGroup(ghc, org, repo, sha)
```

Comment:
would be nice if in the future we changed this to something like `genfiles.NewGroup`

Reply:
Done.